test/src/tests-identity.js

```diff
@@ -3359,6 +3359,80 @@ describe('identity', function() {
         });
     });
 
+    describe('user audiences', function() {
```

Comment:
Is there a reason we are putting these in the identity tests rather than the audience manager tests? They seem to only be testing if feature flag is on or off, so I don't see why this would be in identity?

If anything, maybe we make a feature flags tests file?